help/db_meeting-data.md

```diff
@@ -0,0 +1,39 @@
+### Zawartość tabel
+
+##### Dictionary -- nowe słowniki
+
+| column                   | dictionary_id                        |
+|--------------------------|--------------------------------------|
+| MeetingType.category     | ce12aa23-a5db-49f3-987b-d2db3ab24a3b |
+| Meeting.status           | 3865a3c3-0038-4668-9d55-5d05b79d7fcd |
+| MeetingResource.resource | fdb0f818-aa1e-4ed9-97cc-2a3cb1b702df |
+
+##### MeetingType
+
+| id | facility_id | name               | category_dict_id | default_minutes |
+|----|-------------|--------------------|------------------|-----------------|
+| 1  | 1           | Konsultacja prawna | 1                | 60              |
+| 2  |             | Konferencja        | 2                | 180             |
+
+##### Meeting
+
+| id | facility_in | meeting_type_id | name                  | notes            | date       | start_minutes | minutes | status_dict_id |
```

Comment:
Przez status_dict_id to masz na myśli id z tabeli positions, tak? Bo "dict_id" to jakby znaczy coś innego, brzmi jak "id słownika statusów". No ale pewnie może być jeśli nie chcesz status_position_id albo status_id po prostu.

Reply:
Wydaje mi się, że jest dość istotne, że jak widzę kolumnę status*id, to żebym, wiedział, czy ona się odnosi do tabeli statuses, czy do słownika o tej nazwie

w pracy id ze słownika się nazywa *_name_id, np speciality_name_id, class_name_id, moje *_dict_id wydaje mi się lepsze :D

Reply:
No dobra, ale to jak jest taka konwencja to weź napisz o tym zdanie gdzieś w tym dokumencie.

Podobnie miałeś coś dopisać co znaczy ten + przed niektórymi nazwami, bo napisałeś na discordzie ale nie byłbym przekonany czy umiałbym teraz z pamięci to odtworzyć.